src/data/split_data.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Sun Nov 25 09:45:17 2018
+
+@author: Wanyu Du
+"""
+
+import random
+import os
+
+def split_general_data(input_file, split_ratio, use_shuffle):
+    with open(input_file, 'r', encoding='utf8') as f:
+        lines = f.read().strip().split('\n')
+    print('Read {!s} sentences from {!s}.'.format(len(lines), input_file))
```

Comment:
notice u guys used lots of `format`, maybe you would like to consider `string literal`, which is easier to write in my opinion

https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals

like here can simply be 
```python
f'Read {len(lines)} sentences from {input_file}.'
```

Reply:
OK, I will modify it, thanks!